src/stories/Library/autosuggest-material/autosuggest-material.scss

```diff
@@ -18,6 +18,18 @@
       }
     }
 
+    &.autosuggest__material-item--two {
+      @include breakpoint-m {
+        width: calc(100% / 2);
+      }
+    }
+
+    &.autosuggest__material-item--one {
+      @include breakpoint-m {
+        width: 100%;
+      }
+    }
```

Comment:
Couldn't this have been solved with grid and thereby skipping the need for more classes?